simple/schema.py

```diff
@@ -387,6 +390,22 @@ class Spectra(Base):
         {},
     )
 
+    @validates("access_url", "regime", "source", "telescope", "instrument", "mode")
+    def validate_required(self, key, value):
+        if value is None:
+            raise ValueError(f"Value required for {key}")
```

Comment:
Is it too much too add here that the values must also be in the relevant table? E.g., regime must be in Regimes, telescope in Telescopes, and instrument, mode in Instruments.

Reply:
That validation happens on the actual database ingest with the existing foreign key setup. 
It can't be done in this step because this just checks that you have created a Spectra entry that is valid as far as the python logic goes.